lib/gym/options.rb

```diff
@@ -101,6 +101,16 @@ def self.plain_options
                                      verify_block: proc do |value|
                                        raise "File not found at path '#{File.expand_path(value)}'".red unless File.exist?(value)
                                      end),
+        FastlaneCore::ConfigItem.new(key: :skip_symbols,
+                                     short_option: "-m",
+                                     env_name: "GYM_SKIP_SYMBOLS",
+                                     description: "Should the ipa file *not* include symbols?",
```

Comment:
Any reason for using a negation in the flag name here? Seems confusing, especially when the actual archive flag (`uploadSymbols`) doesn't have a negation. How about `GYM_UPLOAD_SYMBOLS` with default `false`?